setup_config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+environment_name=$(grep "REACT_APP_ENVIRONMENT" packages/oid4vci-demo-frontend/.env | cut -d '=' -f2)
+
+src="packages/agent/conf/docker"
+dest="packages/agent/conf/dev/${environment_name}"
```

Comment:
This for sure should not happen. The docker configs are different from dev configs

Reply:
removed this script